### PR TITLE
Fixed typo of `InterceptorProvider`.

### PR DESCRIPTION
**Pull request by dtaylor1701**
Just a quick fix! Changed `InterceptorProivder` to `InterceptorProvider` in the auth tutorial documentation.

### CODE REVIEW

File: docs/source/tutorial/tutorial-mutations.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ The `ApolloClient` uses something called a `NetworkTransport` under the hood. By
 
 A `RequestChain` runs your request through an array of `ApolloInterceptor` objects which can mutate the request and/or check the cache before it hits the network, and then do additional work after a response is received from the network. 
 
-The `RequestChainNetworkTransport` uses an object that conforms to the `InterceptorProivder` protocol in order to create that array of interceptors for each operation it executes. There are a couple of providers that are set up by default, which return a fairly standard array of interceptors. 
+The `RequestChainNetworkTransport` uses an object that conforms to the `InterceptorProvider` protocol in order to create that array of interceptors for each operation it executes. There are a couple of providers that are set up by default, which return a fairly standard array of interceptors. 
 
 The nice thing is that you can also add your own interceptors to the chain anywhere you need to perform custom actions. In this case, you want to have an interceptor that will add
```
Comment (dtaylor1701) [inline docs/source/tutorial/tutorial-mutations.md:17]: I imagine this is also a typo.

Comment (designatednerd) [reply docs/source/tutorial/tutorial-mutations.md:17]: 🤦‍♀️ less a typo than a dangling thought - thanks for pointing that out. Mind throwing `your token.` in while you're poking around in here?

Comment (dtaylor1701) [reply docs/source/tutorial/tutorial-mutations.md]: Done!

**Review by designatednerd — APPROVED** [review]
Awesome, thanks so much for the fixes!

**Pull request merged**